geode-benchmarks/src/test/java/org/apache/geode/benchmark/tests/PartitionedIndexedAggregateQueryAvgBenchmarkTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.benchmark.tests;
+
+import java.io.File;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junitpioneer.jupiter.TempDirectory;
+
+import org.apache.geode.perftest.TestRunners;
+
+@ExtendWith(TempDirectory.class)
+public class PartitionedIndexedAggregateQueryAvgBenchmarkTest {
+  private File folder;
+
+  @BeforeEach
+  void createTemporaryFolder(@TempDirectory.TempDir Path tempFolder) {
+    folder = tempFolder.toFile();
+  }
+
+  @Test
+  public void benchmarkRunsSuccessfully() throws Exception {
+    PartitionedIndexedAggregateQueryAvgBenchmark test =
+        new PartitionedIndexedAggregateQueryAvgBenchmark();
+    test.setKeyRange(100);
+    test.setQueryRange(10);
```

Comment:
Increase the range; as in this case its indexed, we could increase it for a large number, say 50000 and for non-indexed say 5000. 
For all other tests too.

Reply:
See comment above on sizing the benchmark.

Reply:
> See comment above on sizing the benchmark.

I'm not seeing any comment above about benchmark sizing.

Reply:
Haha, it looks like it didn't save. 

We don't have parametrization support so pick a size for the query that tests what you want to benchmark and nothing more. Picking a large number of results will tend to skew the results heavy on the cost of serialization and GC. Picking a lower number, like 1, focuses on the costs of marshaling the query on the client, transmitting it to the server, running on the server, marshaling the result and sending it back to the client. Performance losses in those areas would be masked by serialization overhead for large results.